include/battle.h

```diff
@@ -278,7 +278,8 @@ struct FieldTimer
 struct WishFutureKnock
 {
     u8 futureSightCounter[MAX_BATTLERS_COUNT];
-    u8 futureSightAttacker[MAX_BATTLERS_COUNT];
+    u8 futureSightBattlerIndex[MAX_BATTLERS_COUNT];
+    u8 futureSightPartyIndex[MAX_BATTLERS_COUNT];
     u16 futureSightMove[MAX_BATTLERS_COUNT];
```

Comment:
I may be mistaken, but wouldn't a slight reorder here realign it so we don't have three u8s and then a u16? Maybe the fact it's an array removes the need, but just a thought